src/_includes/in_page_banner.html

```diff
@@ -1,12 +1,27 @@
-<div class="in-page-banner {{ include.class }}">
+{%- capture classModifier -%}
+  {%- if include.split and include.split == "even" -%}
+    in-page-banner--even
+  {%- elsif include.split and include.split == "right-heavy" -%}
+    in-page-banner--right-heavy
+  {%- endif -%}
```

Comment:
Is it necessary to check `include.split and ...`? Can't we just do `include.split == 'even'`?

Reply:
This is now amended in the following commit [4dae82d](https://github.com/codurance/site/commit/4dae82ddc3b2de1c3c8cfc1efa47b1e6e253355f)